packaging/build/Microsoft.NET.Build.Containers.targets

```diff
@@ -95,4 +97,21 @@
                         EntrypointArgs="@(ContainerEntrypointArgs)"
 						Labels="@(ContainerLabel)"/>
     </Target>
+
+    <Target Name="PublishContainer" DependsOnTargets="$(PublishContainerDependsOn)"
+                                    Condition="'$(MSBuildRuntimeType)' == 'Full'">
+
+        <CreateNewImageToolTask ToolDirectory="$(ContainerizeFolder)"
```

Comment:
Is this line where we'd want to use `NetCoreRoot` like we saw in the other tool tasks?